### PR TITLE
Aswathy/Academy video url special character removed and back button issue

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-  Back navigation when on open video is not working
- Strip down question mark (?) and any other special characters to not show up in the URL when triggering the video popup

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/academy/videos/_all-videos.tsx [hunk 1]
```diff
@@ -40,7 +67,8 @@ const AllVideos = ({ video_data }: AllVideosProps) => {
     const closeVideo = (): void => {
         setShow(false)
         setPlayVideoId('')
-        setTitleParams(undefined)
+        setTitleParams('')
+        queryParams.delete(['t'])
```
Comment (prince-deriv) [inline src/pages/academy/videos/_all-videos.tsx:71]: Can we pass `t` as a string here because your helper accepts both  string and array anyways

Comment (aswathy-deriv) [reply src/pages/academy/videos/_all-videos.tsx:71]: okay

**Comment by sean-binary** [comment]
Playing the video and clicking back takes me to the Academy homepage @aswathy-deriv

File: src/common/utility.js [hunk 2]
```diff
@@ -360,7 +361,52 @@ export const unslugify = (slug) => {
...
+            url.searchParams.delete(key)
+        } else {
+            key.forEach((k) => {
+                url.searchParams.delete(k)
+            })
+        }
+
+        return history.replaceState(null, null, url)
+    },
+}
```
Comment (sean-binary) [inline src/common/utility.js:409]: Is there any difference from the build in browser `new URLSearchParams`?

**Comment by aswathy-deriv** [comment]
> Playing the video and clicking back takes me to the Academy homepage @aswathy-deriv

This is how it is behaved as we are going to the videos page from academy, the history is taken from the academy.
so when u go back from playing the video it takes to the academy page
when u close the video,it stays in the video page

File: src/common/utility.js (hunk 2 above)
Comment (aswathy-deriv) [reply src/common/utility.js:409]: we are using URL.searchParams which is same of URLSearchParams.

File: src/common/utility.js [hunk 3]
```diff
@@ -360,7 +361,53 @@ export const unslugify = (slug) => {
...
+        .replace(/[/]/g, '-') //Replace '/' with single -
+
+export const queryParams = {
+    get: (key) => {
+        const params = new URLSearchParams(location.search)
+        let param_values = {}
+        //To get the params from the url
+
+        if (typeof key === 'string') {
+            console.log('params', params.get(key))
```
Comment (ashraf-deriv) [inline src/common/utility.js:377]: Please remove the console. if it is not necessary

Comment (aswathy-deriv) [reply src/common/utility.js:377]: okay removed.thanks @ashraf-deriv

**Review by sean-binary — APPROVED** [review]

**Pull request merged**